README.md

```diff
@@ -1,248 +1,417 @@
+(*any of the NoSQL datastores which store JSON documents*), it's easier to go "full stack".
+
+See: http://techcrunch.com/2014/11/08/the-rise-and-fall-of-the-full-stack-developer/
+
+### "*Front End*"?
+
+There is ***one (programming) language*** that *all* web
+browsers "understand": ***JavaScript***.  
+As a result it is the _most popular programming language_
+(_see StackOverflow survey results above_!)
+
+Additionally you will need to know two other "languages": HTML and CSS ... but:
+
+![stay calm](https://cloud.githubusercontent.com/assets/194400/21954158/cea1b17a-da41-11e6-9f0e-a73627125934.png)
+
+The web browser or mobile device (displaying content in a "*web view*")
+is referred to as the "*Front End*".
+
+### "Back End"?
+
+When people refer to the "Back End" of an app they are describing the
+"*Web/Application Server*" component(s); don't worry, we will cover all of this
+and it's surprisingly fun!
+
+
+![front-end-back-end](https://cloud.githubusercontent.com/assets/194400/21954171/039c705e-da42-11e6-93d8-7883f365e5fb.png)
+
+There are *many* technologies/languages/frameworks you can use to write
+your server. As stated above, for *simplicity* we prefer to use JavaScript
+(*Node.js*) on the server to keep one programming language across the stack
+and allow us to re-use view code for both server-side and client
+hence the _expression_ "Full-Stack"!
+
+
+## Test Driven Development?
 
-***The Book on Web Application Testing*** is being written one-page-per-day and published at:
+This is where we _differ_ from almost _all_ the other "_beginner_" level books!
+We believe in writing _automated_ tests for all aspects of your application.
+So that is one of the _first_ things you will learn how to do!
 
-> https://leanpub.com/practical-js-tdd
 
-## The Book is Not Ready Yet, Should I Start Reading It?
+## The Book is Not "Finished" Yet, Should I Start Reading It?
 
 The answer is: ***it depends***.
-If you can handle a few typos and want to get started building better software ASAP, then start *now*! If you prefer to wait till its "ready", please star the repo, register your interest at: https://leanpub.com/practical-js-tdd
+If you can handle a few typos and want to get started building better Apps ASAP,
+then start *now*! If you prefer to wait till its "ready", please star the repo,
+and come back later.
 
+## Star the Project :star:
 
-## Spreading the Word >> *Please Re-Tweet*!
+The _easiest_ way you can _contribute_ is
+by "_starring_" this project on GitHub!
+This will help you to "_bookmark_" the content so you can return to it.
+But it will also help the people who "_follow_" you on GitHub to _discover_
+that you find it _interesting_ or _useful_.
+
+The _more_ people star and _share_ the project, the more possible contributors are able to understand the value of contributing and *open sourcing their knowledge*!
```

Comment:
I finished this sentence (by adding what is after the comma), no idea if this was your original intention for the sentence.